src/design-system/typography/typography.ts

```diff
@@ -117,27 +116,3 @@ const createTextSize = ({
 
 export const headingSizes = mapValues(typeHierarchy.heading, createTextSize);
 export const textSizes = mapValues(typeHierarchy.text, createTextSize);
-
-function selectForegroundColors<
-  SelectedColors extends readonly (ForegroundColor | 'accent')[]
->(...colors: SelectedColors): SelectedColors {
-  return colors;
-}
-
-export const textColors = selectForegroundColors(
```

Comment:
As mentioned in an earlier comment, this was removed as it was duplicated in `palettes.ts`.